db/a-schema.sql

```diff
@@ -26,7 +26,7 @@ CREATE TYPE hole AS ENUM (
     'fizz-buzz', 'foo-fizz-buzz-bar', 'fractions', 'happy-numbers',
     'happy-numbers-long', 'intersection', 'isbn', 'kolakoski-constant',
     'kolakoski-sequence', 'leap-years', 'levenshtein-distance',
-    'leyland-numbers', 'look-and-say', 'lucky-tickets', 'morse-decoder',
+    'leyland-numbers', 'look-and-say', 'lucky-numbers', 'lucky-tickets', 'morse-decoder',
```

Comment:
Experimental holes shouldn't update the enum.

Reply:
https://github.com/code-golf/code-golf/pull/610/commits/dd66cffb740ad48d8d3996f3aa26b7d38e6e2c35